bitnami/minio/values.yaml

```
@@ -235,6 +235,7 @@ provisioning:
   ## @param provisioning.enabled Enable MinIO&reg; provisioning Job
   ##
   enabled: false
+  sleepTime: 5
```

You need to add a comment before the parameter. Otherwise, the README automation fails. For example.
```suggestion
  ## @param provisioning.sleepTime Sleep time before checking Minio availability
  ##
  sleepTime: 5
```

---

@dgomezleon ACK and updated